src/features/workflow/WorkflowEditor.tsx

```diff
@@ -71,6 +73,9 @@ const WorkflowEditor: React.FC<WorkflowEditorProps> = ({
   }, [dataset])
 
   const runScript = () => {
+    if (latestDryRunId) {
+      dispatch(removeEvent(latestDryRunId))
+    }
```

Comment:
This should also be cleared if there is a `latestRunId` as well.

This may be for the future, but we should look into if we can combine `latestRunID` and `latestDryRunID`, since we can only have one run at a time, I wonder if there is anything that relies on there being two different run ids.